test/TritonGPU/samples/descriptor-matmul-pipeline.mlir.in

```diff
@@ -1,7 +1,3 @@
-// To regenerate this test case, run the command
-//    triton-opt test/TritonGPU/samples/descriptor-matmul-pipeline.mlir.in -tritongpu-loop-scheduling -tritongpu-pipeline -canonicalize | \
-//    utils/generate-test-checks.py --source test/TritonGPU/samples/descriptor-matmul-pipeline.mlir.in --source_delim_regex="module" \
```

Comment:
Sorry I have a workaround but forgot to push before merging. Just change the default value for `source_delim_regex` so the word `module` doesn't appear in the command line.

Reply:
Yes. See my comments

>  It's also fine to update the regex to something like \bmodule to solve the problem, but I think lifting it from the input file is just simpler.

I think it might be simpler to move it out of the input file, but I'm also OK with keeping it. Let me know what do you think?

Reply:
Oh, I just put it into the docstring of `generate-test-checks.py`

Reply:
IMO it should be easy to find if you see the test failing and don't know why. That's why it's good to have in actual test itself.